internal/auth0/branding/resource.go

```diff
@@ -160,44 +175,31 @@ func updateBranding(ctx context.Context, d *schema.ResourceData, m interface{})
 	return readBranding(ctx, d, m)
 }
 
-func deleteBranding(_ context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+func deleteBranding(_ context.Context, _ *schema.ResourceData, m interface{}) diag.Diagnostics {
 	api := m.(*config.Config).GetAPI()
 
-	if _, ok := d.GetOk("universal_login"); !ok {
-		d.SetId("")
-		return nil
-	}
-
 	if err := checkForCustomDomains(api); err != nil {
-		d.SetId("")
-		return diag.Diagnostics{
-			{
-				Severity: diag.Warning,
-				Summary:  "No custom domains configured",
-				Detail: "Failed to properly destroy the 'auth0_branding' resource " +
-					"because no custom domains are available on the tenant.",
-				AttributePath: cty.Path{cty.GetAttrStep{Name: "universal_login"}},
-			},
+		if err == errNoCustomDomain {
```

Comment:
The delete func was also improved to directly allow the deletion of the resource if there are no custom domains set.